synapse/storage/data_stores/main/push_rule.py

```diff
@@ -13,7 +13,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
this isn't an improvement, honestly, but not a big deal.

Reply:
I think one of the lint tools (isort?) ate that :/, sorry